src/components/FavoritesTab.vue

```diff
@@ -0,0 +1,35 @@
+<script setup lang="ts">
+import { ref } from 'vue'
+import { getPokemon } from '../api.calls'
+import type { Pokemon } from '../api.calls'
+import PokemonResults from './PokemonResults.vue'
+import Button from 'primevue/button'
+
+const props = defineProps<{
+  favorites: Set<number>
+}>()
+
+const clearFavorites = inject<() => void>('clearFavorites', () => {})
+
+const favoritePokemon = ref<Pokemon[]>([])
+
+async function populateFavorites() {
+  try {
+    const response = await Promise.all(
+      Array.from(props.favorites).map((id) => getPokemon(id.toString()))
+    )
+    favoritePokemon.value = response.flatMap((poke) => poke).sort((a, b) => a.id - b.id)
+  } catch (error) {
+    console.log(error)
```

Comment:
A nice touch would be alerting an error message in a toast if the status code isn't 404